manala.apt/vars/main.yml

```diff
@@ -145,6 +145,9 @@ manala_apt_repositories_patterns:
   nodesource_9:
     source: deb https://deb.nodesource.com/node_9.x {{ ansible_distribution_release }} main
     key: nodesource
+  nodesource_10:
+    source: deb https://deb.nodesource.com/setup_10.x {{ ansible_distribution_release }} main
```

Comment:
should be `/node_10.x`, right?

Reply:
Nice catch, but in the [documentation](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions) they use `/setup_10.x`. :thinking: 

I will change for `/node_10.x`